docs/client/components/shared/SocialBar/index.js

```diff
@@ -10,7 +10,7 @@ export default class SocialBar extends Component {
     return (
       <div className={styles.root}>
         <div className={styles.githubButtonWrapper}>
-          <GithubButton user="draft-js-plugins" repo="draft-js-plugins" size="mega" />
+          <GithubButton user="draft-js-plugins" repo="draft-js-plugins" size="large" />
```

Comment:
ok why?

Reply:
According to [github-buttons docs](https://github.com/ntkme/github-buttons#readme) the size parameter can be "None by default or large". Large renders the same as on the current page.

<img width="321" alt="screen shot 2017-09-29 at 11 41 43" src="https://user-images.githubusercontent.com/1851359/31010483-498a98ea-a50b-11e7-83c9-2beb7b170af8.png">